crypto/md5/asm/md5-ia64.S

```diff
@@ -189,9 +189,9 @@ WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.  */
 //		Reads an input block, then calls the digest block
 //		subroutine and adds the results to the accumulated
 //		digest.  It allocates 32 outs which the subroutine
-//		uses as it's inputs and rotating
-//		registers. Initializes the round constant pointer and
-//		takes care of saving/restoring ar.lc
+//		uses as its inputs and rotating registers.
+//		Initializes the round constant pointer and takes
```

Comment:
_Note: 'takes' is unchanged, except for whitespace changes around it. Seems to be a display error._

Reply:
md5-ia64.S doesn't exist anymore on 1.1.1 anyway